packages/cli/src/command-build.js

```diff
@@ -167,11 +167,20 @@ async function executeDuringPhase(
   biltin,
   dryRun,
 ) {
-  return await buildPackages(
-    finalPackagesToBuild,
-    makePackageBuild(jobConfiguration, rootDirectory, buildOptions, biltin),
-    dryRun,
-  )
+  /**@return {import('@bilt/build').BuildPackageFunction} */
+  const buildCounter = () => {
+    let pointer = 1
```

Comment:
What is the purpose of this variable?

Reply:
this is the variable in the closer - for increment it - do you have another suggestion?